File: client/src/app/_dtos/hr/feedbackDto.ts
export interface IFeedbackDto
{
    id:number;
    customerId:number;
    customerName:string;
    city:string;
    officialName:string;
    email:string;
    phoneNo:string;
    dateReceived: Date;
    dateIssued:Date;
    gradeAssessedByClient:string;
    customerSuggestion: string;
}

export class FeedbackDto
{
    id=0;
    customerId=0;
    customerName='';
    city='';
    officialName='';
    email='';
    phoneNo='';
    dateReceived=new Date();
    dateIssued=new Date();
    gradeAssessedByClient='';
    customerSuggestion='';
}

File: api/DTOs/HR/FeedbackDto.cs
namespace api.DTOs.HR
{
    public class FeedbackDto
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public string CustomerName { get; set; }
        public string City { get; set; }
        public string Email { get; set; }
        public string PhoneNo { get; set; }
        public DateTime DateReceived { get; set; }
        public DateTime DateIssued { get; set; }
        public string GradeAssessedByClient { get; set; }
        public string CustomerSuggestion { get; set; }
    }
}